### ISSUE TITLE
Announcement: New maintainer

### ISSUE THREAD

**Issue by NearHuscarl** [NearHuscarl/flutter_login#108]
Hello everyone, I'm here to announce that @juliansteenbakker will be the new maintainer of this project.

@juliansteenbakker has experience maintaining the other high-profile package [qr_code_scanner](https://github.com/juliuscanute/qr_code_scanner). He'll fix some long standing bugs here and maybe add some highly requested features in the issues, we'll see. However please keep in mind that, many of the maintainers in the open source space do this for free in their free time which is not always available, things may be slow so please be patient with them.

I'd like to thank @juliansteenbakker so much for helping me out on this one. Good luck @juliansteenbakker, I hope everything goes well for you.

**Comment by juliansteenbakker** [comment]
Thank you very much! I hope I can help to make this package get even better, together with the support of other contributors.

**Comment by lcsvcn** [comment]
Thank you, @NearHuscarl! Welcome, @juliansteenbakker.

So great to hear that this AWESOME widget will be kept alive and updated.

Count me in to help in anything you need :)

**Comment by juliansteenbakker** [comment]
@lcsvcn If you want, i can add you as reviewer of pull requests for upcoming pull-requests. Thank you for your help!

**Comment by Programmer9211** [comment]
@juliansteenbakker I also want to do contributions on this project :)

**Comment by juliansteenbakker** [comment]
@Programmer9211 Thank you for your time. It would be great if you can check #138 for any errors or faults.

**Comment by lcsvcn** [comment]
@juliansteenbakker Yes, that would be nice. Count me in! I am working when I can to help to improve this project, since I use this in my projects. If you want to add me to help you maintain.

I have currently a PR open with a new feature and I will be opening another one soon to refactor some technical improvements, like code modularization that I noticed when I was implementing the userType feature.

I have marked with a **TODO** comment in my code and I will work this in another PR to help keep my PR as little as possible to help review.

And sorry for the delay to answer, I have not received any notification for this message :(
